ledger/controllers/surveyor.js

```diff
@@ -36,10 +37,45 @@ const server = async (request, reply, runtime) => {
     surveyor.payload = entry.payload
     surveyor.parentId = entry.parentId
   }
+  return surveyor && addChoices(runtime, surveyor)
+}
 
+async function addChoices (runtime, surveyor) {
+  const payload = surveyor.payload || {}
+  surveyor.payload = payload
+  const adFree = payload.adFree || {}
+  payload.adFree = adFree
+  adFree.choices = await getAdjustedChoices(runtime, 'USD', ['BAT'])
   return surveyor
 }
 
+async function getAdjustedChoices (runtime, base, currencies) {
+  const rates = await runtime.currency.rates(base, currencies)
+  return underscore.mapObject(rates, choicesPrices)
+}
+
+function choicesPrices (ratio) {
+  const basePrice = (new BigNumber(1)).dividedBy(ratio)
+  const table = [
+    [3, 5, 7, 10, 20],
+    [4, 6, 9, 12, 25],
+    [5, 8, 11, 17, 35],
+    [6, 10, 14, 20, 40],
+    [9, 12, 20, 35, 50],
+    [15, 25, 35, 50, 100],
+    [20, 35, 50, 85, 175],
+    [30, 50, 70, 100, 200]
+  ]
```

Comment:
We may want to run the values in the table by the product team before shipping if we haven't already

Reply:
agreed. they came from @mandar-brave perhaps he can paste the spreadsheet he provided here

Reply:
https://docs.google.com/spreadsheets/d/1-X4lWEJBZLlYlJu9_2vhkx0PSjsC2OsBohNl7-N4d-c/edit#gid=0

Reply:
I feel like we should probably just define the relation between the price options so we could scale this list no matter what the price of BAT is.  We're kind of limiting ourselves to only the options in the hard coded table and we'd have to come back and add more options if prices went way up or way down.  Do you have thoughts on the matter?

I'm good with how you have it now since these are the approved values from the product team, and since it seems by this comment https://github.com/brave-intl/bat-ledger/pull/395#discussion_r260564322 we'll have a hard cap amount on the higher BAT amounts anyway.

Reply:
yeah, i agree that we shouldn't be hard coding. in this case we're just kind of stuck between nice round values vs decimals and the estimations that round, man made values produce. I would like to move away from fiat as the base and move toward thinking about things in terms of BAT, even if the price does fluctuate, but that would take an effort from all areas of the business that i didn't think was worth delaying the feature over. I do think it's a discussion worth having. there are other companies that have stopped